### PR TITLE
Feature blui 3294 empty state api docs

### PR DESCRIPTION
**Pull request by jeffvg**
Fixes # BLUI-3294.

#### Changes proposed in this Pull Request:
- add empty state api doc
- 
- 

#### To Test:
- yarn start:reactdev
- navigate to Empty State -> empty state api docs

### CODE REVIEW

**Comment by jeffvg** [comment]
prettier is looping for me locally. Any idea why?

**Review by huayunh — COMMENTED** [review]
Change looks good. There might be a few clean ups required after #471 is merged though.

**Review by bkarambe — COMMENTED** [review]
Please resolve the merge conflicts

File: docs/src/markdown/EmptyState.mdx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        }}
+        component="img"
+        alt="Empty State Image"
+        src={EmptyStateImage}
+    />
+</Box>
+
+## Empty State API
+
+<Box
```
Comment (bkarambe) [inline docs/src/markdown/EmptyState.mdx:27]: As markdownMapping has been merged to base branch, which will also cover styling  for tables in mdx, so we wont need sx prop here. Just keeping Box around the table works fine.

Comment (jeffvg) [reply docs/src/markdown/EmptyState.mdx:27]: so all this isn't needed now?

image

Comment (huayunh) [reply docs/src/markdown/EmptyState.mdx:27]: yep

**Comment by jeffvg** [comment]
looks like the channel value api doc was merged before the style sheet updates. I'll update this in the PR

File: docs/src/markdown/EmptyState.mdx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Any other props supplied will be provided to the root element (`div`).
+
+### Style Overrides
+
+You can override the default styles used by Brightlayer UI by:
+
+-   using the `sx` prop
+-   passing a `classes` prop with keys from the `Name` column below
+-   using the `Global CSS Class` in your main stylesheet
```
Comment (ektasawant) [inline docs/src/markdown/EmptyState.mdx:47]: As per our last PR comments need to remove `` from `Global CSS Class`.
```suggestion
-   using the Global CSS Class in your main stylesheet
```

Comment (jeffvg) [reply docs/src/markdown/EmptyState.mdx:47]: ok did not know this. In the latest stylesheet PR thats merged we have 6 mdx files for drawer that are still using back-quotes here. I'll open issue or story to revisit all the previous api doc files

**Comment by jeffvg** [comment]
cleared up some additional changes in here, so this is ready for a review

**Review by huayunh — APPROVED** [review]
lgtm

**Pull request merged**